docs/source/community.rst

```diff
@@ -26,7 +26,7 @@ We recommend discussing new features in form of a proposal on the issue tracker
 
 All code contributed via pull request needs to adhere to the following guidelines:
 
-  1. Most signac packages follow the `git-flow`_ branching model.
+  1. Most signac packages follow the `OneFlow`_ branching model.
```

Comment:
Why "most"? Are there exceptions? If it's held up by signac-dashboard, I'd prefer to say "The signac framework follows ..." because I'll change it over to OneFlow next release.

Reply:
Yes, I didn't know about dashboard. I can switch to all if that's your intention.